clifford/test/test_function_cache.py

```diff
@@ -0,0 +1,20 @@
+import numpy as np
+from numba import generated_jit
+import warnings
+
+# Make the test fail on a failed cache warning
+warnings.simplefilter("error")
+
+
+@generated_jit(cache=True)
+def foo(x):
+    from clifford.g3c import e3
```

Comment:
```suggestion
    from clifford.g3 import e3
```
May as well make this test faster :)